src/blockdata/block.rs

```diff
@@ -400,7 +400,7 @@ mod tests {
     // Check testnet block 000000000000045e0b1660b6445b5e5c5ab63c9a4f956be7e1e69be04fa4497b
     #[test]
     fn segwit_block_test() {
-        let segwit_block = include_bytes!("../../test_data/testnet_block_000000000000045e0b1660b6445b5e5c5ab63c9a4f956be7e1e69be04fa4497b.raw").to_vec();
+        let segwit_block = include_bytes!("../../tests/data/testnet_block_000000000000045e0b1660b6445b5e5c5ab63c9a4f956be7e1e69be04fa4497b.raw").to_vec();
```

Comment:
While you're at it I think you could remove `.to_vec()`.

Reply:
Done, thanks.

Reply:
Turns out this is needed for rust 1.29 unless we want to cast in the `assert_eq` statement at bottom of function. To be honest I find Rust 1.29 extremely annoying, sometimes I can build with it other times I cannot so I just left this as is.

FTR I'm using this
```
build-with-rust-1.29 () {
    cargo clean
    rm -f Cargo.lock
    cargo +1.29 generate-lockfile --verbose && cargo +1.29 update -p cc --precise "1.0.41" --verbose
    cargo +1.29 build --features=use-serde
}
```
But it only works sometimes, other times I get errors like this
```
Updating cc v1.0.72 -> v1.0.41
error: unable to get packages from source

Caused by:
failed to parse manifest at `/home/tobin/.cargo/registry/src/github.com-1ecc6299db9ec823/proc-macro2-1.0.36/Cargo.toml`

Caused by:
editions are unstable

Caused by:
feature `edition` is required

this Cargo does not support nightly features, but if you
switch to nightly channel you can add
`cargo-features = ["edition"]` to enable this feature
``` 
Any tips most appreciated.

Reply:
oh I worked it out, I needed 
```
    cargo +1.29 generate-lockfile --verbose && \
        cargo +1.29 update -p cc --precise "1.0.41" --verbose && \
        cargo +1.29 update --package "serde" --precise "1.0.98" --verbose && \
        cargo +1.29 update -p "serde_derive" --precise "1.0.98" --verbose
```